readme.md

```diff
@@ -100,6 +102,34 @@ snapchat
 stock market
 ```
 
+## Tumblr
```

Comment:
Bringing up identity politics is being used as of late as a way to silence discussions, and it's been seeping lately into the dev circles.
Came across a discussion from some highly popular tweeters recently that basically went like this:
A: "I have opinion X"
B: "That opinion is wrong because Y"
A: "How typical - another white male mansplaining how my opinion is wrong"
B: "o_O"